pkg/okteto/pipeline.go

```diff
@@ -58,6 +61,7 @@ func (c *OktetoClient) DeployPipeline(ctx context.Context, name, repository, bra
 			"branch":     graphql.String(branch),
 			"variables":  variablesVariable,
 			"filename":   graphql.String(filename),
+			"source":     graphql.String(origin),
 		}
 
 		err := mutate(ctx, &mutation, queryVariables, c.client)
```

Comment:
Should we check if the mutation returns the new error `"Unknown argument \"source\" on type \"GitDeploy\"")` so we can run the deprecated version of `deployPipeline`

Reply:
yes, I think we should do it because the `source` parameter was released on the OE 0.10.7 and probably there are still OE in previous versions. @pchico83 Can you confirm is there is any OE in version < 0.10.7?

Reply:
yes, this change should be backward compatible. One option is to pass this value as an envvar of the pipeline deployment, not sure if that makes things easier or not

Reply:
Got it. If we use the deprecatedDeploy tho it would send all deploys on 0.10.7 to the deprecated path of 0.10.0, right? I'll try to send it as an envvar as Pablo suggests